src/filesystem.lisp

```diff
@@ -11,53 +11,64 @@
 ;;;; time, we have to somehow "walk" their reference to copy them for
 ;;;; a "resource bundle" that can then be distributed.
 
-(defparameter *subpath* nil "Directory subpath to be added to each of the below. Must end with /. Useful for testing.")
+(defun emotiq/user/root/ ()
+  "The root of all user persistence"
+  (let ((d (if (not (emotiq:production-p))
+               (asdf:system-relative-pathname :emotiq "../var/etc/") ;; TODO rename other than etc
+               #+:windows
```

Comment:
@guitarvydas @echupriyanov @easye
Should use functional interfaces in `uiop:os-mumble` instead of compile time conditionals.

Reply:
Fixed.